src/Common/CancelToken.cpp

```diff
@@ -112,7 +112,7 @@ bool CancelToken::wait(UInt32 * address, UInt32 value)
     // Start cancelable wait. Spurious wake-up is possible.
     futexWait(address, value);
 
-    // "Release" futex and check for cancelation
+    // "Release" futex and check for cancellation
```

Comment:
(Totally fine to change back to "cancellation", just note that there is a spelling whitelist in utils/check-style/aspell-ignore/en/aspell-dict.txt)